experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -384,7 +384,7 @@ describe("editable-tree: editing", () => {
 	});
 
 	for (const [fieldDescription, fieldKey] of testCases) {
-		describe(`can create, edit and delete ${fieldDescription}`, () => {
+		describe(`can create, edit, move and delete ${fieldDescription}`, () => {
```

Comment:
I don't like relying on large complicated tests like this. It might be a good ideal to keep this change to this test, but we should also have a dedicated unit tests testing the various cases of move (move left and right in a single field and move to another field at least).

Reply:
added unit tests for moving left/right within same field and moving to another field